docs/team/nknguyenhc.md

```diff
@@ -12,11 +12,11 @@ Given below are my contributions to the project.
 * **New Feature**: Add an email to a person.
 The command adds the new email to the target person's current list of emails.
 
-* **New Feature**: Attach a link to a person.
-The link can be the person's social page or personal website.
-The command adds the new link to the target person's current list of emails.
+* **New Feature**: Attach a webLink to a person.
+The webLink can be the person's social page or personal website.
+The command adds the new webLink to the target person's current list of emails.
 
-* **Code contributed**: [RepoSense link]()
+* **Code contributed**: [RepoSense webLink]()
```

Comment:
Please make sure that you do not change `link` to `webLink` in the markdown files.

Reply:
Done!